.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+<!--
+  Thanks for submitting a pull request!
+  We appreciate you spending the time to work on these changes. Please provide enough information so that others can review your pull request.
+
+  Before submitting a pull request, please make sure the following is done:
+
+  1. Fork [the repository](https://github.com/avantifellows/quiz-backend) and create your branch from `master`.
+  2. Run the installation steps from the project's [README.md](https://github.com/avantifellows/quiz-backend#readme).
+  3. Please ensure coding standard and conventions are followed. You can find the details at https://vuejs.org/v2/style-guide/#Priority-A-Rules-Essential-Error-Prevention.
```

Comment:
The backend is not in vue right?

Reply:
I remember linking PEP8 ka style guide here. No idea why that is not the case :/